README.md

```diff
@@ -37,12 +37,13 @@ yarn add react-native-device-info
 ```shell
 react-native link react-native-device-info
 ```
-
 (or using [`rnpm`](https://github.com/rnpm/rnpm) for versions of React Native < 0.27)
 
 ```shell
 rnpm link react-native-device-info
 ```
+*For IOS*
+You still need to run `pod install` after running the above link command inside your `IOS` folder.
```

Comment:
I like the location - this is the exact spot I think - and it's almost perfect for me, but since this section is not Pod-specific and we still provide direction for pod or non-pod, could you say *for iOS users using Pods* ? Then it's merge time :-) - thank you!

Reply:
K will do that